flow-server/src/main/java/com/vaadin/flow/server/frontend/NodeUpdater.java

```diff
@@ -401,6 +401,7 @@ static void addVaadinDefaultsToJson(JsonObject json) {
         defaults.put("webpack-manifest-plugin", "3.0.0");
         defaults.put("@types/validator", "13.1.0");
         defaults.put("validator", "13.1.17");
+        defaults.put("mkdirp", "^1.0.4");
```

Comment:
```suggestion
        defaults.put("mkdirp", "1.0.4");
```
We should be strict on the versions and not allow random updates.

Reply:
Thought so too initially, but in case there is a new version and the platform itself would depend on it, then it might be that we force an older version that has bugs ? Neither option seems good to me, not sure which sucks less

Reply:
Platform should be able to override defaults. As they only override older versions if there exists a marking.